test_task_1.py

```diff
@@ -0,0 +1,28 @@
+# Our goal is to create a fullname field by concatenating a first and last name
+import unittest
+import task_1
+import random
+
+
+class TestFullNameCreated(unittest.TestCase):
+    def setUp(self) -> None:
+        self.user_records = task_1.create_full_name()
+        self.record_to_test = random.choice(self.user_records)  #avoid wasting time testing every record
+        self.split_name = self.record_to_test["full_name"].split()
+        self.first_half = self.split_name[0]
+        self.last_half = self.split_name[-1]
+
+    def test_full_name_field_exists(self) -> None:
+        self.assertTrue(self.record_to_test["full_name"])
```

Comment:
Perhaps a more accurate way to test the field existing would be:
```
self.assertTrue("full_name" in self.record_to_test)
```
Using this approach literally tests the existence of the key in the dictionary.

Reply:
I like this method you're suggesting much more! I'll apply it.